namecheap.py

```diff
@@ -15,6 +16,10 @@
 }
 NAMESPACE = "http://api.namecheap.com/xml.response"
 
+# default values for the retry mechanism
```

Comment:
Sure thing! ... shall I provide additional commits to this PR, or do you have another preference?

(if you haven't guessed, I'm totally easy :-) ... and deferential to your design/goals for PyNamecheap)